way_to_home/user_profile/views.py

```diff
@@ -1 +1,46 @@
 """This module that provides base logic for CRUD of user profile`s model objects."""
+
+from django.http import JsonResponse
+from django.views import View
+from utils.responsehelper import (RESPONSE_200_UPDATED,
+                                  RESPONSE_400_EMPTY_JSON,
+                                  RESPONSE_400_DB_OPERATION_FAILED,
+                                  RESPONSE_400_OBJECT_NOT_FOUND,)
+
+
+class UserProfileView(View):
+    "Class that handles HTTP requests for user_profile model."""
+
```

Comment:
You should add `http_method_names = ['get', 'put']`

Reply:
you are right, thanks